### DISCUSSION TITLE
Github Proble

### DISCUSSION THREAD

**Discussion by MohamedMehrez** [community/community#136267] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question about account problems

### Body

I have an account that I use for work as an intern and I had my email compromised and suddenly i can't acces my github and it's not visable for a week now i send a ticket to github support and no one answered since last tuesday and my boss is angry with me and is considering ending my internship for not being able to push the code and they are not letting me use another account their word is to settle this fast so pls can anyone help me all i want to do is to get back to work and to get my account back and change the email linked to it 
Ticket is submitted: #2938424
pls someone answer me and contact me in this email : ezynomics934@outlook.com

**Comment by queenofcorgis** [comment, marked as answer]
Hi @MohamedMehrez, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note:  You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
